SU-SWS/stanford_news
# D8 core 1900: News byline fixes

# READY FOR REVIEW

# Summary
- Fixes to the conditionals for the author, date and pipe and the social media icons.

# Needed By (Date)
- ASAP

# Urgency
- high

# Steps to Test

1. pull in this branch
2. Look at 3 different articles. One with date and author, one with date only and one with author only.
3. Verify it looks ok in various window widths.

# Affected Projects or Products
- News Accelerant.

# Associated Issues and/or People
- D8CORE-1900

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
